### PR TITLE
docs: changelog for 4.2.3

### PR DESCRIPTION
**Pull request by MuyangAmigo**

### CODE REVIEW

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 4.2.3 - Feb 17, 2023
+
+Incremental version for Teams Toolkit with multiple bug fixes.
+
+- Fixed an unexpected string change in the create new app flow.
```
Comment (sffamily) [inline packages/vscode-extension/CHANGELOG.md:7]: create new app ->  create-new-app

File: packages/vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+- Fixed an issue where there's no response when click on the Get Started button.
```
Comment (sffamily) [inline packages/vscode-extension/CHANGELOG.md:8]: click -> clicking
Get Started -> "Get Started"

File: packages/vscode-extension/CHANGELOG.md (hunk 1 above)
Comment (MuyangAmigo) [reply packages/vscode-extension/CHANGELOG.md]: Updated

File: packages/vscode-extension/CHANGELOG.md (hunk 2 above)
Comment (MuyangAmigo) [reply packages/vscode-extension/CHANGELOG.md]: Updated.

**Pull request merged**